targets/targets.json

```diff
@@ -1325,7 +1325,7 @@
     "KW24D": {
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M4",
-        "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
+        "supported_toolchains": ["ARMC5", "GCC_ARM", "IAR"],
```

Comment:
We are seeing runtime failures with this target with ARMC6. This needs further investigation/partner support, so currently KW24D will stay with ARMC5.

Reply:
Can we have an issue to track this?

Reply:
@0xc0170 - Please see ARM Internal Ref: IOTCORE-1059